lib/buildenvsetup/ceconan.ts

```diff
@@ -175,12 +175,20 @@ export class BuildEnvSetupCeConanDirect extends BuildEnvSetupBase {
                 encoding: null,
             };
 
-            request(packageUrl, settings)
+            // https://stackoverflow.com/questions/49277790/how-to-pipe-npm-request-only-if-http-200-is-received
+            const req = request(packageUrl, settings)
                 .on('error', error => {
                     logger.error(`Error in request handling: ${error}`);
                     reject(error);
                 })
-                .pipe(gunzip);
+                .on('response', async res => {
```

Comment:
I think `async` isn't required here, no harm done by leaving it though

Reply:
Good spot. removing